static/js/popovers.js

```diff
@@ -302,8 +302,13 @@ exports.show_user_profile = function (user) {
     $("#user-profile-modal-holder").html(templates.render("user_profile_modal", args));
     ui.set_up_scrollbar($("#user-profile-modal #body"));
     $("#user-profile-modal").modal("show");
-
     settings_account.intialize_custom_user_type_fields("#user-profile-modal #content", user.user_id, false, false);
+
+    $(".modal-backdrop, #user-profile-modal").on('click', function (e) {
```

Comment:
How did you pick this selector?  It's not obvious to me why `#user-profile-modal` belongs here; are you trying to make clicking on the modal itself close the backdrop?

Reply:
That is a very good point.  This was actually to keep the compose box open when clicking the close 'x' as well as clicking outside the modal. Changing this to focus in on the ```close``` of ```user-profile-modal``` would be a better approach, correct?